core-js/src/main/javascript/cdf/components/ccc/BaseCccComponent.js

```diff
@@ -199,7 +199,13 @@ define([
         });
       }
 
-      this.chart.render();
+      if (!!this.cleanElementBeforeRender) {
+        this.chart.render();
+      } else {
+        this.chart.render({
+          bypassAnimation: true
+        });
```

Comment:
Idem about the `!!`.
Why the difference in treatment of renders?
I understand the motivation, but it should only be applied on renders after the first one.
Note that `cleanElementBeforeRender` to `true` should be the new default (but we cannot change it now). Like this, animations would never be performed.
The `animate` option should be specified as `false` from the start in these use cases.